consolelabs/mochi-api
# feat: add floor price to search coin

**What does this PR do?**

-   [x] Add floor price to search coin api response

**How to test**

-  GET `defi/coins` for searching token api
-  POST `defi/watchlist` for add token to wl api

-   What are the edge cases that you might see in this PR?

**Media (Loom or gif)**
<img width="1451" alt="CleanShot 2023-05-31 at 14 02 53@2x" src="https://github.com/consolelabs/mochi-api/assets/14150687/0be5e7bb-4c60-4644-a54a-2a97b81ea0a0">

<img width="1463" alt="CleanShot 2023-05-31 at 14 00 36@2x" src="https://github.com/consolelabs/mochi-api/assets/14150687/a3440448-2680-4fbc-8cb6-d7d542ab299d">
